azkaban-webserver/src/main/resources/azkaban/webapp/servlet/velocity/projectsidebar.vm

```diff
@@ -16,7 +16,7 @@
 
           <div class="well" id="project-sidebar">
             <h3>$project.name</h3>
-            <p class="editable" id="project-description">$project.description</p>
+            <p class="editable" id="project-description">$esc.html($project.description)</p>
```

Comment:
That's what I am hoping for. See my todo comment above.

Even if that is possible, we can't just turn it on without risk regression. It's better to do it separately IMHO.